### PR TITLE
feat: make proposal sidebar sticky

### LINKED ISSUE #250: Make the proposal sidebar sticky
https://user-images.githubusercontent.com/467712/162525572-2043eae7-21c1-4056-9e1f-7a30285a1559.mov

### PR DESCRIPTION
**Pull request by andyesp**
Closes #250

### CODE REVIEW

**Review by 1emu — APPROVED** [review]
📜 ✅

File: src/pages/proposal.tsx [hunk 1]
```diff
@@ -262,86 +262,88 @@ export default function ProposalPage() {
...
+                      loading={updatingStatus}
+                      style={{ width: '100%' }}
+                      onClick={() => patchOptions({ confirmStatusUpdate: ProposalStatus.Passed })}
+                    >
+                      {t('page.proposal_detail.pass')}
+                    </Button>
+                    <Button
+                      basic
+                      loading={updatingStatus}
+                      style={{ width: '100%' }}
```
Comment (ncomerci) [inline src/pages/proposal.tsx:335]: I think it's a good opportunity to move these `style={{ width: '100%' }}` to a css class, what do you think? There are a bunch on this page 😬

**Review by ncomerci — APPROVED** [review]

File: src/pages/proposal.tsx (hunk 1 above)
Comment (andyesp) [reply src/pages/proposal.tsx:335]: Maybe we can create an issue to remove inline styles

Comment (andyesp) [reply src/pages/proposal.tsx:335]: Fixed in #310

**Pull request merged**